torch/testing/_benchmark.py

```diff
@@ -0,0 +1,46 @@
+from typing import Callable, List, Union
+
+import torch
+
+
+def benchmark_func(
+    func: Callable,
```

Comment:
Nit: type with ParamSpec for args and TypeVar for Return Type

Reply:
But I don't know those, because number of parameters depends on the function, same as number of return arguments

Reply:
The point of ParamSpec is that infers all the arguments / kwargs from the Callable though? And setting the ReturnArg to a TypeVar does the same.